assignment.md

```diff
@@ -11,63 +11,76 @@ If the version control concepts or `git` commands are still confusing to you, co
            Date:   Tue Mar 2 23:25:21 2021 -0500                                                                                                                               Made sensitive_info file where I'll stupidly add fake sensitive information lol                                                                                                                                                             commit dfc7f82392bf551a6d0715402f5edaddc7bb29aa                                 Author: Amanda Simon <a.simon3210@gmail.com>                                    Date:   Tue Mar 2 23:08:09 2021 -0500                                                                                                                               Answered up to question 5                                                   Date:   Tue Mar 2 23:28:01 2021 -0500                                                                                                                               Added fake SSN, credit card number and routing number. Pls rob me hackers :)                                                                                commit 39baecc95ed04145bcba7e31029f6b15e491b5a3                                 Author: Amanda Simon <a.simon3210@gmail.com>                                    Date:   Tue Mar 2 23:25:21 2021 -0500                                                                                                                               Made sensitive_info file where I'll stupidly add fake sensitive information lol                                                                                                                                                             commit dfc7f82392bf551a6d0715402f5edaddc7bb29aa                                 Author: Amanda Simon <a.simon3210@gmail.com>                                    Date:   Tue Mar 2 23:08:09 2021 -0500                                                                                                                               Answered up to question 5                                                                                                                                   commit e439dc1da8733a1899c4b5b0e550c596dc806964                                 Author: Amanda Simon <a.simon3210@gmail.com>                                    Date:   Mon Mar 1 22:32:28 2021 -0500                                                                                                                               Answered first two questions                                                                                                                                commit d8f643c65b4e8ceb9e3a792699c803432aacfb62 (origin/main, origin/HEAD, main)Author: Xihe Xie <axiezai@gmail.com>                                            Date:   Mon Feb 15 11:10:27 2021 -0500                                                                                                                              improved instructions   
```

Comment:
You want to do the entire command from the link: 

```
git filter-branch --force --index-filter \
  "git rm --cached --ignore-unmatch PATH-TO-YOUR-FILE-WITH-SENSITIVE-DATA" \
  --prune-empty --tag-name-filter cat -- --all
```

This will remove your history where the sensitive info is added, and filters the `git diff` in the history so your password or something doesn't show. A new commit should appear where this happens. I will post about this in detail in slack.